cmd/veneur/main.go

```diff
@@ -187,6 +189,15 @@ func main() {
 	if err != nil {
 		logger.WithError(err).Fatal("Could not initialize server")
 	}
+
+	go func() {
```

Comment:
+1 to @rma-stripe's comments. Also note that we _don't_ vendor this file. I would recommend moving this logic into a sub-package and creating a static function for collecting base metrics, so we can call that here, and vendor that in.

Reply:
Gave it a try, let me know what you think.